libmscore/style.h

```diff
@@ -1096,6 +1096,99 @@ enum class Sid {
       STYLES
       };
 
+// Internal resolution factor that multiplies values originally in points
+static constexpr qreal DPI_F = 5; // screen resolution factor
+
+// Unit conversion factors
+static constexpr qreal INCH  = 25.4;  // millimeters per inch
+static constexpr qreal PICA  = 12.0;  // points per pica and didot per cicero
+static constexpr qreal DIDOT = 1.06574601373228; // didot per point
+static constexpr qreal PPI   = 72.0;  // PostScript points per inch
+static constexpr qreal DPI   = DPI_F * PPI; // 360.0 dots per inch
+static constexpr qreal DPMM  = DPI / INCH;  //  14.173228346 dots per millimeter
+
+// Default spatium values: MuseScore and MusicXML
+// Both values are equivalent: 25.0 @360dpi == 10.0 @144dpi == 5pt
+static constexpr qreal SPATIUM20   = DPI_F * (DPI / PPI); // 25.0 == 5pt == DPI_F * DPI_F
+static constexpr qreal SPATIUM_XML = 10.0;  // MusicXML, in demi-points @144dpi
+
+//------------------------------------------------------------------------------
+//   PageUnit
+//    Qt provides only an enumeration for units: QPageLayout::Unit
+//    MuseScore needs all this additional data per unit. The data is for:
+//     a) Display and widget control settings for pagesettings.ui
+//     b) Contents for the units drop-down lists in pagesettings and prefsdialog
+//     c) Unrounded unit conversions, because QPageLayout rounds everything
+//     d) _key is a lookup key and the value written to the .mscx file
```

Comment:
This comment applies to one of my next PRs in this batch.  Please ignore it for now.